src/components/shared/TractionFieldGroup.vue

```diff
@@ -37,7 +37,6 @@
       :is="component"
       v-if="component"
       :id="fieldId"
-      v-model="getAttribute"
```

Comment:
v-bind="$attrs" Is this another way of doing it , where we can avoid getAttribute computed method. But  please check which is the better way as I am not sure which is more appropriate.

Reply:
I think you know more than me in this area so happy to go with whatever you think. The issue with this v model method was that it couldn't 'set' values because it was a computed method. But not sure if its needed at all. If we bind attrs can we remove the value prop as well or does it just replace value?

Reply:
As my understanding v-bind="$attrs" is like fall through attribute so we don't need to  explicitly declared in the receiving component's props especially in cases where we need all props to bind directly to the wrapped component. For e.g if a button is wrapped in div and we want all fallthrough attributes like class and v-on listeners to be applied to the inner <button>, not the outer <div>. We can achieve this with inheritAttrs: false and v-bind="$attrs"